app/views/SelectedUsersView/index.tsx

```diff
@@ -0,0 +1,188 @@
+					<HeaderButton.Container>
+						<HeaderButton.Item
+							title={users.length > 0 ? buttonTextHeader : I18n.t('Skip')}
+							onPress={nextActionHeader}
+							testID='selected-users-view-submit'
+						/>
+					</HeaderButton.Container>
+				)
+		};
+		navigation.setOptions(options);
+	}, [users.length, maxUsers]);
+
+	useEffect(() => {
+		if (isGroupChat()) {
+			dispatch(addUser({ _id: user.id, name: user.username, fname: user.name as string }));
+		}
+	}, []);
+
+	useEffect(() => {
+		let querySubscription: Subscription;
+		const init = async () => {
+			try {
+				const db = database.active;
+				const observable = await db.get('subscriptions').query(Q.where('t', 'd')).observeWithColumns(['room_updated_at']);
+
+				querySubscription = observable.subscribe(data => {
+					const chats = orderBy(data, ['roomUpdatedAt'], ['desc']) as ISelectedUser[];
+					setChats(chats);
+				});
+			} catch (e) {
+				log(e);
+			}
+		};
+		init();
+
+		return () => {
+			dispatch(reset());
+			if (querySubscription && querySubscription.unsubscribe) {
+				querySubscription.unsubscribe();
+			}
+		};
+	}, [dispatch]);
+
+	const handleSearch = useCallback(async (text: string) => {
+		const result = await searchMethod({ text, filterRooms: false });
+		setSearch(result);
+	}, []);
+
+	const toggleUser = (userItem: ISelectedUser) => {
+		// Disallow removing self user from the direct message group
+		if (isGroupChat() && user.username === userItem.name) {
+			return;
+		}
+
+		if (!isChecked(userItem.name)) {
+			if (isGroupChat() && users.length === maxUsers) {
+				return showErrorAlert(I18n.t('Max_number_of_users_allowed_is_number', { maxUsers }), I18n.t('Oops'));
+			}
+			logEvent(events.SELECTED_USERS_ADD_USER);
+			dispatch(addUser(userItem));
+		} else {
+			logEvent(events.SELECTED_USERS_REMOVE_USER);
+			dispatch(removeUser(userItem));
+		}
+	};
+
+	const _onPressItem = (item = {} as ISelectedUser) => {
+		if (item.search) {
+			toggleUser({ _id: item._id, name: item.username as string, fname: item.name });
+		} else {
+			toggleUser({ _id: item._id, name: item.name, fname: item.fname });
+		}
+	};
+
+	const searchOrChats = (search.length > 0 ? search : chats) as ISelectedUser[];
+	// filter DM between multiple users
+	const data = searchOrChats.filter(sub => !isGroupChatMethod(sub));
+
+	return (
+		<SafeAreaView testID='select-users-view'>
+			<StatusBar />
+			<FlatList
+				data={data}
+				keyExtractor={item => item._id}
+				renderItem={({ item, index }) => {
+					const name = useRealName && item.fname ? item.fname : item.name;
+					const username = item.search ? (item.username as string) : item.name;
+					let style = { borderColor: colors.separatorColor };
+					if (search.length > 0 && index === search.length - 1) {
+						style = { ...style, ...sharedStyles.separatorBottom };
+					}
+					if (search.length === 0 && index === chats.length - 1) {
+						style = { ...style, ...sharedStyles.separatorBottom };
+					}
```

Comment:
We talked about this.
Can't you use ListFooterComponent with a small condition?
It seems unnecessary to test this on every item.